deployment/deploy/3000-deploy-VestingRegistry.js

```diff
@@ -0,0 +1,25 @@
+const path = require("path");
+const { getContractNameFromScriptFileName } = require("../helpers/utils");
+const { sendWithMultisig } = require("../helpers/helpers");
+const { deployWithCustomProxy } = require("../helpers/helpers");
+const col = require("cli-color");
+//const deploymentName = getContractNameFromScriptFileName(path.basename(__filename));
+const func = async function (hre) {
```

Comment:
missing setting admin manager role on deployment/upgrade - should be included into the contract's initialization